constitution.tex

```diff
@@ -0,0 +1,328 @@
+	\item in the case of a second tie between the same candidates, another round of voting must occur before the close of the General Meeting, with only the tied candidates appearing on the ballot;
+	\item in the case of a third tie between the same candidates, the winner shall be determined by sortition; and
+	\item any informality or irregularity with the elections must be brought to the attention of the Clubs and Societies Administration Officer within fourteen (14) days of the election.
+	\end{enumerate}
+\item The election of Delegates of the Management Committee without a General Meeting shall take place in the following manner:
+	\begin{enumerate}
+	\item the Management Committee shall be at liberty to nominate any member of the Society to serve as a Delegate of the Management Committee;
+	\item the Management Committee must give every member of the Society an opportunity to vote secretly and securely;
+	\item the members of the Society shall have no less than one week to cast their vote;
+	\item the vote of every member of the Society shall be of equal weight;
+	\item the nominee will be elected as a Delegate of the Management Committee if and only if the number of votes of the members of the Society in favour equals or exceeds the number of members of the Society needed to constitute quorum of a General Meeting of the Society and the number of votes of the members of the Society in favour equals or exceeds twice the number of votes of the members of the Society against.
+	\item any informality or irregularity with the elections must be brought to the attention of the Clubs and Societies Administration Officer within fourteen (14) days of the election.
+	\end{enumerate}
+\end{enumerate}
+
+\section{Resignation or Removal from Office of Membership of Management Committee}
+\begin{enumerate}
+\item Any member of the Management Committee may resign from membership of the Management Committee at any time by giving notice in writing to the Secretary. Such resignation shall take effect at the time such notice is received by the Secretary, unless a later date is specified in the notice, where it shall take effect on that later date.
+\item The Secretary may resign in the same manner as any other member of the Management Committee, with the exception of written notice shall instead be given to the President.
+\item If a member of the Management Committee:
+	\begin{enumerate}
+	\item fails to comply with any of the provisions of the Constitution;
+	\item has membership fees in arrears;
+	\item violates the Society's Code of Conduct;
+	\item acts in a manner considered to be injurious or prejudicial to the character of interests of the Society;
+	\item ``yeet"s excessively;
+	\item ``\includegraphics[height=1em]{y.png}"s excessively; or
+	\item dabs excessively;
+	\item deletes the \texttt{:dab:} emoji; or
+	\item uses \textit{Comic Sans}
```

Comment:
While I appreciate the humour, not sure if these items should be in the constitution